### PR TITLE
CP-29084: Rebranding XenServer to Citrix Hypervisor in Toolstack

### PR DESCRIPTION
**Pull request by michael2012z**
This PR is to change the branding from "XenServer" to "Citrix Hypervisor" in the title and header of SDK API reference pages.

Signed-off-by: Michael Z <michael.zhao@citrix.com>

### CODE REVIEW

**Comment by kc284** [comment]
Purely for the SDK, there are also the following instances which I believe should change:
- ocaml/doc/basics.md:3:This document defines the XenServer Management API....
- a few instances of "XenServer" and "XenAPI" in API member docs in datamodel_*.ml (not in the datamodel_types.ml though as this is legacy branding which we keep).
- ocaml/doc/index.html:36: <h1 class="title">XenAPI Classes</h1> (=> just "API")

Beyond the SDK, and as Christian mentioned above, there are some more instances of XenServer in the scripts (mostly in echoed messages) - shouldn't these change?
Also, what about those two: ocaml/xapi/vpx.ml:57,64 ?

(I'd also recommend changing several instances of xenserver in code comments to generic "server" simply so we get into the right mindset and consequently avoid exposing it accidentally when we write user visible messages).

**Comment by kc284** [comment]
By the way, since we are changing the above, we can't avoid updating the READMEs in the SDK.

**Comment by michael2012z** [comment]
Hi, @kc284 , @lindig , I modified this PR to cover the rebranding of Toolstack, including SDK and some other parts that are visible to users in toolstack.

**Comment by lindig** [comment]
Looking good to me.

File: ocaml/doc/index.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
 			<div id="header"></div>
 			<div id="main">
 				<div id="container"><div id="content">
-					<h1 class="title">XenAPI Classes</h1>
+					<h1 class="title">Management API Classes</h1>
```
[the remark this answers is not in the document]
Comment (michael2012z) [reply ocaml/doc/index.html]: Reasonable!

**Comment by michael2012z** [comment]
Comments handled.

**Pull request merged**